src/server/commands.rs

```diff
@@ -124,11 +123,37 @@ pub enum Command {
         path: String,
     },
     /// The `LIST` command
+    // This command causes a list to be sent from the server to the
+    // passive DTP.  If the pathname specifies a directory or other
```

Comment:
DTP?

Reply:
This comes verbatim from the spec :-) RFC 959